Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM adzerk/boot-clj:latest as builder
+
+ENV BOOT_VERSION=2.7.2
+ENV BOOT_CLOJURE_VERSION=1.8.0
+
+RUN boot uber
+
+FROM openjdk:jre-alpine
+
+ARG VERSION
+
+WORKDIR /usr/local/github-changelog
+
+COPY target/github-changelog-$VERSION.jar ./github-changelog.jar
```

Comment:
I believe you will need to use the `--from=builder` argument to copy from the builder image.

Reply:
You are right, it should be fixed now